src/index.html

```diff
@@ -1,14 +1,37 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="menu">
+
+      <a class="logo" href="#home">
+        <img src="moyo-logo.png" alt="moyo-header"
+        height="40px" width="40px">
+      </a>
```

Comment:
1. due to bem it's better to use class `header__logo`
2. something is wrong with formatting, you can use prettier to fix it, if you have more then 2 attributes every attribute should be from new line
```suggestion
      <a class="logo" href="#home">
        <img 
          src="moyo-logo.png" 
          alt="moyo-header"
          height="40px" 
          width="40px"
        >
      </a>
```